src/redux/conference-schedule.js

```diff
@@ -0,0 +1,159 @@
+    return function (dispatch) {
+        api({
+            uri: '/conference/schedule/' + day
+        }, function (err, body) {
+            if (err) {
+                dispatch(module.exports.setDayError(err));
+                dispatch(module.exports.setScheduleError(err));
+                return;
+            }
+
+            if (typeof body !== 'undefined') {
+                var columns = body.columns;
+                var rows = body.rows || [];
+                // Group events by the time period in which they occur (for presentation)
+                var scheduleByChunk = rows.reduce(function (prev, cur) {
+                    var cleanedRow = {};
+                    for (var i = 0; i < columns.length; i++) {
+                        if (cur[i].length > 0) {
+                            cleanedRow[columns[i]] = cur[i];
+                        }
+                    }
+                    cleanedRow['uri'] = '/conference/' + cleanedRow.rowid + '/details';
+                    var chunk = cleanedRow.Chunk;
+                    if (typeof prev.chunks[chunk] === 'undefined') {
+                        prev.chunks[chunk] = [cleanedRow];
+                        prev.info.push({
+                            name: chunk,
+                            time: cleanedRow.Start
+                        });
+                    } else {
+                        prev.chunks[chunk].push(cleanedRow);
+                    }
+                    return prev;
+                }, {chunks: [], info: []});
+                // group periods of time by start time
+                scheduleByChunk.info.sort(function compare (a, b) {
+                    var aAm = (a.time.substr(a.time.length - 1, a.time.length) === 'a') ? true : false;
+                    var bAm = (b.time.substr(b.time.length - 1, b.time.length) === 'a') ? true : false;
+                    var aTime = parseInt(a.time.substr(0, a.time.length - 1));
+                    var bTime = parseInt(b.time.substr(0, b.time.length - 1));
+
+                    if (aTime < bTime) {
+                        if (bAm && !aAm) {
+                            return 1;
+                        } else {
+                            return -1;
+                        }
+                    } else {
+                        if (aAm && !bAm) {
+                            return -1;
+                        } else {
+                            return 1;
+                        }
+                    }
+                });
+                var schedule = [];
+                for (var i = 0; i < scheduleByChunk.info.length; i++) {
+                    schedule.push({
+                        info: scheduleByChunk.info[i],
+                        items: scheduleByChunk.chunks[scheduleByChunk.info[i].name]
+                    });
+                }
+                dispatch(module.exports.setDay(day));
+                dispatch(module.exports.setSchedule(schedule));
```

Comment:
Since these are so intertwined, I think you could actually leave this as a single reducer, and have day and the schedule  details be two parts of the same state.